layouts/shortcodes/prismjs.html

```diff
@@ -0,0 +1,27 @@
+{{ $inner := replaceRE "^\n" "" .Inner }}
+{{ if len .Params | eq 0 }}
+  <pre><code>{{ $inner }}</code></pre>
+{{ else }}
+  {{ if .IsNamedParams }}
+    <pre
+      class="
+      {{ if .Get "lang" }}language-{{ .Get "lang" }} {{ end }}
+      {{ if .Get "line-numbers" }}line-numbers {{ end }}
```

Comment:
```suggestion
      {{ if .Get "line-numbers" }} line-numbers{{ end }}
```

Reply:
I'd add space at the beginning of the class, because it's additional element, so the best option here is to add whitespace only when it's needed, like: 
1. one class: `class="language-css"`
2. two classes `class="language-css[ line numbers]"`
3. three classes `class="language-css[ line numbers][ command-line]"`